MAPL_Base/MAPL_HistoryGridComp.F90

```diff
@@ -857,12 +866,12 @@ subroutine Initialize ( gc, import, dumexport, clock, rc )
 ! Check if field list had a vector pair and collection marked as conservative
 ! Abort if this is the case as this is not supported
 ! ---------------------------------------------------------------------------
-       if (associated(list(n)%field_set%vector_list)) then
-          if (list(n)%conservative /= 0) then
-             call WRITE_PARALLEL('Can not have a vector pair in a conservative collection')
-             _ASSERT(.false.,'needs informative message')
-          end if
-       end if
+!@@       if (associated(list(n)%field_set%vector_list)) then
+!@@          if (list(n)%conservative /= 0) then
+!@@             call WRITE_PARALLEL('Can not have a vector pair in a conservative collection')
+!@@             _ASSERT(.false.,'needs informative message')
+!@@          end if
+!@@       end if
```

Comment:
Again if this code should not be used, please delete it (and the preceding comment.

Reply:
Agree. This block could be removed altogether...